tools/debug/analyze_map.py

```diff
@@ -20,13 +25,16 @@
 from bark.runtime.viewer.matplotlib_viewer import MPViewer
 from bark.core.geometry import Point2d, Polygon2d
 from bark.core.world.opendrive import XodrDrivingDirection
+import bark
+
+bark.core.commons.GLogInit("", "./", 5, True, "")
 
 import numpy as np
 import itertools
 
 # Name and Output Directory
 # CHANGE THIS #
-map_name = "DR_CHN_Merging_ZS_partial_v04_lower_edited"
+map_name = "left_turn_rural"
```

Comment:
use map that is publicly available, so one can e.g. visualize the lane corridor and understand the concept